04_TS_decorator/helpers/helpers.ts

```diff
@@ -0,0 +1 @@
+export type Actions = "getStudentsList" | "getYoungestStudentAge" | "assignStudents" | "graduateStudents"
```

Comment:
Please follow this condition:
```
> Параметр `action` метода `execute()` декторатора типизируйте с помощью ключевого слова `keyof`. При этом данный параметр должен быть типизирован таким образом, чтобы иметь значения ТОЛЬКО 4 строк: названий указанных выше методов.
```

Reply:
Please follow the task: you shouldn't create any additional type for the `action` parameter typing. Only the `University` class should be used.